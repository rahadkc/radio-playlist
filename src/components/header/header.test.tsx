import { cleanup, render, screen } from '@testing-library/react'
import * as React from 'react'
import Header from './'

describe('<Footer> component', () => {
  const title = 'Station List'
  afterEach(cleanup)

  test('should render Header component with title prop', () => {
    render(<Header title={title} />)

    expect(screen.getByText(title)).toBeInTheDocument()
    expect(screen.queryByText(title)).toMatchSnapshot()
  })
})
