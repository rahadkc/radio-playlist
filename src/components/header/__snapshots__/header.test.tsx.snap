// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> component should render Header component with title prop 1`] = `
<div
  class="title"
>
  Station List
</div>
`;
